Sources/Models/PublicEpisodes/0067-SwiftUIAndStateManagementPt3.swift

```diff
@@ -795,6 +835,23 @@ Unfortunately, it came with a _lot_ of boilerplate, but let's make sure it actua
   ),
   Episode.TranscriptBlock(
     content: """
+This episode was recorded with Xcode 11 beta 3. While it allowed you to derive `Binding`s of sub-state, a bug prevented it from propagating mutations over presentation boundaries, like navigation links and modal sheets. This bug has since been fixed in Xcode 11 beta 5, and state is now much more composable.
+
+```
+var state: Binding<(favoritePrimes: [Int], activityFeed: [AppState.Activity])>
+""",
+    timestamp: nil,
+    type: .correction
+  ),
+  Episode.TranscriptBlock(
+    content: """
+Let's make sure it works. First, we can replace the object binding in our favorite primes view.
```

Comment:
is this new transcript block supposed to be here?

Reply:
It was combined above, but I wanted to break it in half with the correction so that it more directly applied to the previous text.

Reply:
ah I see, should that part of the text be removed from the other transcript block then?

Reply:
Dang, I swear I did that. Xcode's been crashing a bunch for me.